### PR TITLE
Add docker image tests to build.sh

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: ci/build.sh [hunk 1]
```diff
@@ -2,55 +2,88 @@
...
-  echo "Pushing docker image $DOCKER_TAG:$version..."
-  echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-  docker push $DOCKER_TAG:$version
-}
+  echo
+  echo "Running docker image tests..."
 
-if [ -n "$DOCKER_PASSWORD" ] && [ -n "$TRAVIS_TAG" ]; then
-  push_docker_image "$VERSION"
+  for app in ${apps}; do
```
Comment (sumo-drosiek) [inline ci/build.sh:50]: what does the shellcheck says for that?

Comment (pmalek-sumo) [reply ci/build.sh:50]:
```
shellcheck --enable all --exclude SC2154 ci/build.sh; echo $?
0
```

File: src/go/cmd/k8s-api-test/main.go [hunk 2]
```diff
@@ -13,32 +13,47 @@
...
+	log.Printf("Kubernetes version: %s", version.GitVersion)
 
-    pods, err := clientset.CoreV1().Pods("").List(metav1.ListOptions{Limit: 20})
-    handleErr("Failed when fetching list of pods", err)
+	pods, err := clientset.CoreV1().Pods("").List(metav1.ListOptions{Limit: 20})
+	handleErr("Failed when fetching list of pods", err)
 
-    log.Printf("Received data for %d pods in the cluster\n", len(pods.Items))
+	log.Printf("Received data for %d pods in the cluster\n", len(pods.Items))
 }
```
Comment (sumo-drosiek) [inline src/go/cmd/k8s-api-test/main.go:59]: What is the desire format in terms of spaces vs tabs?

Comment (pmalek-sumo) [reply src/go/cmd/k8s-api-test/main.go:59]: I'd say https://golang.org/doc/effective_go.html#formatting

tldr: use tabs

Comment (pmalek-sumo) [reply src/go/cmd/k8s-api-test/main.go:59]: This file already used tabs (on the top of the file for instance) in some places and spaces elsewhere.

**Review by sumo-drosiek — APPROVED** [review]
